Makefile

```diff
@@ -69,11 +69,11 @@ endif
 	docker-compose up -d --remove-orphans
 	$(call php-0, apk add --no-cache graphicsmagick $(ADD_PHP_EXT))
 	$(call php-0, kill -USR2 1)
-	$(call php, composer global require -o --update-no-dev --no-suggest "hirak/prestissimo:^0.3")
```

Comment:
It must be duplicated, not removed here

Reply:
btw as build will move to composer 2 (released today) this extension no longer needed